### PR TITLE
Filecheck

### PR DESCRIPTION
**Pull request by ahirag**
fix port range
add home path variable
delete garabage

### CODE REVIEW

File: nginx.py [hunk 1]
```diff
@@ -23,7 +24,7 @@
 	#Creating all site's configuration in config.d folder
 	config = io.open(nginx_config_path+str(i) + '.conf', 'w')
 	for line in io.open('template.conf', 'r'):
-		line = line.replace('$port_num', str(i))
+		line = line.replace('$port_num', str(port_range+i))
```
Comment (eliranKimi) [inline nginx.py]: Whats the point in port_range variable?
we use i as the indicator for the port .

for now its 1-3
but we can change it 1800-1803

File: nginx.py [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+
+home_folder = os.getenv("HOME")
+port_range = 18000;
 
 for i in range(1,3):
 
 	#path="/data/www/"+str(i)
-	path = "/home/eliran/nginx-shit/test/"+ str(i)+"/"
-	nginx_config_path= "/home/eliran/nginx-shit/test/"
+	path = home_folder+ "/nginx-shit/test/"+ str(i)+"/"
```
Comment (eliranKimi) [inline nginx.py]: Wer'e not creating the configuration files in our home folder.
its supposed to be in a different path. (i just used it for testing)

File: nginx.py [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
+	nginx_config_path= home_folder+ "/nginx-shit/test/"
```
Comment (eliranKimi) [inline nginx.py]: Same as comment above

File: nginx.py (hunk 2 above)
Comment (ahirag) [inline nginx.py]: yea i know but the home folder was neccesary variable in order to me run it at my workstation

File: nginx.py (hunk 1 above)
Comment (ahirag) [inline nginx.py]: yea just changed it to be same as his mission :)

File: nginx.py (hunk 3 above)
Comment (ahirag) [inline nginx.py]: cheerz mate

**Comment by eliranKimi** [comment]
מהמהמה

**Pull request closed without merging**